public/js/photo-new.js

```diff
@@ -0,0 +1,346 @@
+/*
+ * This script will handle all common javascript functions within the photo
+ * module.
+ * Depends: jquery
+ */
+
+Photo = {
+    vote: function(item) {
+        if (item.voted) {
+            return;
+        }
+
+        var url = $('a[href="' + item.src + '"]').data('vote-url');
+        $.post(url, function(data) {
+            $('.pswp__button--like')
```

Comment:
Besides the temporary tooltip, also provide the user some visual feedback. A quick way to achieve this is to add some a highlighting color.

```
$(...).css({"color": "#D40026"})
```

Reply:
![Screenshot 2019-09-30 at 22 07 08](https://user-images.githubusercontent.com/7968933/65912570-b3c04080-e3ce-11e9-9ef7-63d4cb58fbc0.png)